src/test/java/org/takes/facets/auth/social/PsLinkedinTest.java

```diff
@@ -67,9 +70,32 @@ public void logins() throws IOException {
         final Take take = new TkFork(
             new FkRegex(
                 "/uas/oauth2/accessToken",
+                // @checkstyle AnonInnerLengthCheck (100 lines)
                 new Take() {
                     @Override
                     public Response act(final Request req) throws IOException {
+                        MatcherAssert.assertThat(
+                            new RqPrint(req).printBody(),
+                            Matchers.stringContainsInOrder(
+                                Arrays.asList(
+                                    "grant_type=authorization_code",
+                                    new StringBuffer("client_id=")
+                                        .append(lapp)
+                                        .toString(),
+                                    "redirect_uri=",
+                                    new StringBuffer("client_secret=")
```

Comment:
@super132 don't you think that `String.format()` is more suitable here? see http://www.yegor256.com/2014/06/19/avoid-string-concatenation.html